iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/tripping/TrippingTopologyTraverser.java

```diff
@@ -41,6 +41,8 @@ public TraverseResult traverse(Terminal terminal, boolean connected) {
                         terminalsToDisconnect.add(terminal);
                     }
                     return TraverseResult.TERMINATE_PATH;
+                } else if (affectedTerminals != null) {
+                    affectedTerminals.add(terminal);
```

Comment:
Maybe we should only fill traversed terminal in case of bus/breaker topo. Even if in that case we will have the same list of terminal than in terminalsToDisconnect, but that would more consistent.

Reply:
done